proposals/simd/portable-simd.md

```diff
@@ -0,0 +1,1324 @@
+the lanes. Floating point comparisons follow IEEE semantics:
+
+```python
+def S.lessThan(a, b):
+    def lt(x, y):
+        return x < y
+    return S.lanewise_comparison(lt, a, b)
+```
+
+### Less than or equal
+* `s8x16.lessThanOrEqual(a: v128, b: v128) -> b8x16`
+* `s16x8.lessThanOrEqual(a: v128, b: v128) -> b16x8`
+* `s32x4.lessThanOrEqual(a: v128, b: v128) -> b32x4`
+* `s64x2.lessThanOrEqual(a: v128, b: v128) -> b64x2`
+* `u8x16.lessThanOrEqual(a: v128, b: v128) -> b8x16`
+* `u16x8.lessThanOrEqual(a: v128, b: v128) -> b16x8`
+* `u32x4.lessThanOrEqual(a: v128, b: v128) -> b32x4`
+* `u64x2.lessThanOrEqual(a: v128, b: v128) -> b64x2`
+* `f32x4.lessThanOrEqual(a: v128, b: v128) -> b32x4`
+* `f64x2.lessThanOrEqual(a: v128, b: v128) -> b64x2`
+
+```python
+def S.lessThanOrEqual(a, b):
+    def le(x, y):
+        return x <= y
+    return S.lanewise_comparison(le, a, b)
+```
+
+### Greater than
+* `s8x16.greaterThan(a: v128, b: v128) -> b8x16`
+* `s16x8.greaterThan(a: v128, b: v128) -> b16x8`
+* `s32x4.greaterThan(a: v128, b: v128) -> b32x4`
+* `s64x2.greaterThan(a: v128, b: v128) -> b64x2`
+* `u8x16.greaterThan(a: v128, b: v128) -> b8x16`
+* `u16x8.greaterThan(a: v128, b: v128) -> b16x8`
+* `u32x4.greaterThan(a: v128, b: v128) -> b32x4`
+* `u64x2.greaterThan(a: v128, b: v128) -> b64x2`
+* `f32x4.greaterThan(a: v128, b: v128) -> b32x4`
+* `f64x2.greaterThan(a: v128, b: v128) -> b64x2`
+
+```python
+def S.greaterThan(a, b):
+    def gt(x, y):
+        return x > y
+    return S.lanewise_comparison(gt, a, b)
+```
+
+### Greater than or equal
+* `s8x16.greaterThanOrEqual(a: v128, b: v128) -> b8x16`
+* `s16x8.greaterThanOrEqual(a: v128, b: v128) -> b16x8`
+* `s32x4.greaterThanOrEqual(a: v128, b: v128) -> b32x4`
+* `s64x2.greaterThanOrEqual(a: v128, b: v128) -> b64x2`
+* `u8x16.greaterThanOrEqual(a: v128, b: v128) -> b8x16`
+* `u16x8.greaterThanOrEqual(a: v128, b: v128) -> b16x8`
+* `u32x4.greaterThanOrEqual(a: v128, b: v128) -> b32x4`
+* `u64x2.greaterThanOrEqual(a: v128, b: v128) -> b64x2`
+* `f32x4.greaterThanOrEqual(a: v128, b: v128) -> b32x4`
+* `f64x2.greaterThanOrEqual(a: v128, b: v128) -> b64x2`
+
+```python
+def S.greaterThanOrEqual(a, b):
+    def ge(x, y):
+        return x >= y
+    return S.lanewise_comparison(ge, a, b)
+```
+
+## Load and store
+
+Load and store operations are provided for `v128` vectors, but not for the
+boolean vectors; we don't want to impose a bitwise representation of the boolean
+vectors.
+
+The memory operations work on an abstract `Buffer` instance which can be
+addressed by a `ByteOffset` type. Unaligned memory operations are allowed, but
+they may be slower than aligned operations.
+
+This specification does not address bounds checking and trap handling for memory
+operations. It is assumed that the range `addr .. addr+15` are valid offsets in
+the buffer, and that computing `addr+15` does not overflow the `ByteOffset`
+type. Bounds checking should be handled by the embedding specification.
```

Comment:
WebAssembly simply states: "Out of bounds accesses trap. If the access is a store, if any of the accessed bytes are out of bounds, none of the bytes are modified."

We should do the same here.